scripts/test-migrations.sh

```diff
@@ -31,8 +38,17 @@ function run_tests_for_version() {
     echo "===== Finished testing version $VERSION ====="
 }
 
+if [ "$1" == "mysql" ]; then
+    echo "===== Testing MySQL ====="
+    # run a mysql instance in docker
+    docker run --name mysql -d -p 3306:3306 -e MYSQL_ROOT_PASSWORD=password mysql:latest
+
+    # mysql takes a while to start up
+    sleep 30
+fi
+
 # List of versions to test
-VERSIONS=("0.40.0" "0.40.3" "0.41.0" "0.43.0" "0.44.1" "0.44.3" "0.45.2" "0.45.3" "0.45.4" "0.45.5" "0.45.6" "0.46.0" "0.47.0")
+VERSIONS=("0.40.0" "0.40.3" "0.41.0" "0.43.0" "0.44.1" "0.44.3" "0.45.2" "0.45.3" "0.45.4" "0.45.5" "0.45.6" "0.46.0" "0.47.0" "0.50.0")
```

Comment:
Are we going to update this list as we go? Maybe it is better to fetch it from Pypi, excluding yanked ones, and take the last patch version of each minor release?

Reply:
This script is just a placeholder that we update as we go, I think. Going forward (i.e. planned for 2024) we'll have a separate script for each different migration path we want to test (since once you go pre-0.40.0 it gets complicated with dependencies + the pipeline syntax).